packages/fluentui/react-northstar/src/components/Avatar/Avatar.tsx

```diff
@@ -24,6 +25,9 @@ export interface AvatarProps extends UIComponentProps {
    */
   accessibility?: Accessibility<never>;
 
+  /** Avatar can contain icon. */
```

Comment:
Please should we add that icon is only rendered if `image` is not present here or in the docs example?